lib/danger/danger_core/plugins/dangerfile_github_plugin.rb

```diff
@@ -92,6 +92,14 @@ def self.instance_name
       "github"
     end
 
+    # @!group PR Review
+    # Current PR Review object.
+    # @return [GitHubReview]
+    #
```

Comment:
@orta looks cool 👍 
The small feedback: 
1. `@return [GitHubReview]` -> `@return [GitHubSource::Review]` or we can just leave `Review`.
2. Should we mention `github.review.markdown(...)` ?